qiskit_nature/drivers/second_quantization/electronic_structure_driver.py

```diff
@@ -37,9 +37,9 @@ class MethodType(Enum):
     UKS = "uks"
 
 
-class FermionicDriver(BaseDriver):
+class ElectronicStructureDriver(BaseDriver):
```

Comment:
I think the deprecation of `qiskit_nature.drivers.fermionic_driver.FermionicDriver` is still missing. Same for the `BosonicDriver` on that level.

Reply:
I added the deprecations